src/Microsoft.SqlTools.ServiceLayer/Admin/Database/DatabasePrototype.cs

```diff
@@ -730,6 +744,8 @@ public bool HasSameValueAs(DatabaseData other)
             {
                 bool result =
                     (this.recoveryModel == other.recoveryModel) &&
+                    (DateTime.Compare(this.lastBackupDate, other.lastBackupDate) == 0) &&
```

Comment:
It isn't necessary to use the `DateTime.Compare`, you should be able to just use `==`